### ISSUE TITLE
Ignore node_modules when linting

### ISSUE THREAD

**Issue by mrsimpson** [graywolf336/temporary-rocketlets-dev-environment#3]
When re-using a node module, one does not have control over the packages while implementing the Rocketlet.
The dev-environment refusing the build due to linting errors, particularly in dependencies is not satisfying.

```
[09:53:41] 'lint-no-exit-ts' errored after 3.77 s
[09:53:41] Error in plugin 'gulp-tslint'
Message:
  /.../temporary-rocketlets-dev-environment/rocketlets/smarti/node_modules/http-status-enum/src/index.ts [43, 35]: Missing trailing comma.

```

**Comment by graywolf336** [comment]
Ah, so your folder structure is:

```
dev-env
  -> rocketlets
     -> smarti (which is a git repository)
        -> node_modules
```

Okay, so the `rocketlets/**/node_modules` needs to be ignored :+1:

**Comment by mrsimpson** [comment]
Exactly. That's how I thought you intended a dev environment to be used. 
Any other ideas, how it should be consumed?

Additionally, it was nice it the .gitignore ignored all the Rocketlets - except the samples. Don't know whether such expressions are supported though

**Comment by graywolf336** [comment]
Yeah it is, I just wanted to confirm it.

Yes, I am going to see about that as I think it is possible. :+1:
